### PR TITLE
:memo: Fixing of docs

### PR DESCRIPTION
**Pull request by Endercheif**
### Changes

-   `adds`: missing middleware 
-   `fixed`: the docs (mostly), errors the occurred, incorrect links
-   `improvements`: ...

 ### Check off the following

-   [x] I have tested my changes with the current requirements
-   [x] My Code follows the pep8 code style.

### CODE REVIEW

**Comment by Endercheif** [comment]
Link to (mostly) fixed docs.
https://pincer-test.readthedocs.io/en/fix-docs

there will be another pr on Wednesday. I did this to minimize conflicts as more is added.

File: pincer/__init__.py [hunk 1]
```diff
@@ -51,8 +51,8 @@ class VersionInfo(NamedTuple):
 
     def __repr__(self) -> str:
         return f"{self.major}.{self.minor}.{self.micro}" + (
-            f"-{self.release_level}{self.serial}"
-            * (self.release_level is not None)
+                f"-{self.release_level}{self.serial}"
+                * (self.release_level is not None)
```
Comment (trag1c) [inline pincer/__init__.py:55]: why

File: pincer/middleware/guild_members_chunk.py [hunk 2]
```diff
@@ -19,12 +19,12 @@ async def guild_member_chunk_middleware(self, payload: GatewayDispatch):
...
     ----------
-    payload : :class:`GatewayDispatch`
+    payload : :class:`~pincer.core.dispatch.GatewayDispatch`
         The data received from the guild member chunk event.
 
     Returns
     -------
-    Tuple[:class:`str`, List[:class:`~pincer.objects.events.guild.GuildMemberChunkEvent`]]
+    Tuple[:class:`str`, List[:class:`~pincer.objects.events.guild.GuildMembersChunkEvent`]]
         ``on_guild_member_chunk`` and a ``GuildMemberChunkEvent``
```
Comment (trag1c) [inline pincer/middleware/guild_members_chunk.py:28]: Shouldn't L28 also change here?

File: pincer/objects/guild/guild.py [hunk 3]
```diff
@@ -421,9 +421,10 @@ async def from_id(cls, client: Client, _id: Union[int, Snowflake]) -> Guild:
     async def get_member(self, _id: int) -> GuildMember:
         """|coro|
         Fetches a GuildMember from its identifier
+
         Parameters
         ----------
-        _id:
+        _id: int
```
Comment (trag1c) [inline pincer/objects/guild/guild.py:427]: Shouldn't it be `` :class:`int` ``?

Comment (Endercheif) [reply pincer/objects/guild/guild.py:427]: It works and is more concise 
https://pincer-test.readthedocs.io/en/fix-docs/pincer.objects.guild.html#pincer.objects.Guild.get_member

File: pincer/__init__.py (hunk 1 above)
Comment (Endercheif) [reply pincer/__init__.py:55]: I think either black did it or my IDE.

**Review by Lunarmagpie — APPROVED** [review]
GatewayDspatch looks like it caused a lot of pain.

**Comment by Endercheif** [comment]
> GatewayDspatch looks like it caused a lot of pain.

🥲 fun times

**Review by trag1c — APPROVED** [review]

**Review by RPSMain — APPROVED** [review]
Lgtm might properly review in the morning if not already merged by then

**Comment by Lunarmagpie** [comment]
I'm ok with merging this even though sigma hasn't approved

**Review by Sigmanificient — APPROVED** [review]

**Pull request merged**